src/index.js

```diff
@@ -801,7 +801,7 @@ function contextifyLogger(context, logger) {
         logger[methodName] = function (message, ...meta) {
             // logger.info('ok', { pid: 1 }) becomes:
             // logger.info('ok', { pid: 1, context: { ... } })
-            meta[0] = { ...meta[0], context };
+            meta[0] = Object.assign(meta[0], context);
```

Comment:
hmm beware that here you are mutating the meta object.

`Object.assign({}, meta[0], context)` ?

Reply:
outch